packages/storybook/stories/PowerPoint Live.stories.mdx

```diff
@@ -0,0 +1,77 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta
+  id="ppt-live"
+  title="Concepts/PowerPoint Live"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
```

Comment:
_Creating a thread to talk about the screenshot used_

ACS UI lib has a horizontal scroll bar in your screenshot. Can we take a new screenshot with it wider to remove the horizontal scroll bar?
![image](https://github.com/Azure/communication-ui-library/assets/97130533/5aae2870-b689-4c90-8c94-2e9062a2812d)

Reply:
Also, can we not add the top URL bar for the acs app? The site link is visible on your screenshot. So essentially, just cut the top part off of the browser.